src/Compilers/Core/Portable/Operations/ITupleBinaryOperation.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    /// <summary>
+    /// Represents a comparison of two operands that returns a bool type.
+    /// <para>
+    /// Current usage:
+    ///  (1) C# tuple binary operator expression.
+    /// </para>
+    /// </summary>
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
+    /// </remarks>
+    public interface ITupleBinaryOperation : IOperation
```

Comment:
This interface needs to be reviewed by the IOperation design group. @jinujoseph, can you add this to the list of things to talk about Wednesday?